addons/mail/models/mail_thread.py

```diff
@@ -1269,8 +1269,8 @@ def _message_route_process(self, message, message_dict, routes):
                 # Report failure/record success of message creation except if alias is not defined (fallback model case)
                 try:
                     thread = ModelCtx.message_new(message_dict, custom_values)
-                except Exception:
-                    if alias:
+                except Exception as e:
+                    if alias and (not e.args or e.args[0] != 'MAIL_ANSWER_ALREADY_SENT'):
                         with self.pool.cursor() as new_cr:
                             self.with_env(self.env(cr=new_cr)).env['mail.alias'].browse(alias.id
                             )._alias_bounce_incoming_email(message, message_dict, set_invalid=True)
```

Comment:
Would either use `raise Exception("Nice Error Message") from Exception("TECHNICAL_ERROR_CODE")` and then fetch the "special value" from `Exception.__cause__` so we can have nicer logs.

Or use a specific new Exception class and catch it before catching `Exception`
Specific exception could even hold values to pass to `_alias_bounce_incoming_email` so we can create a regular bounce email with a custom body (can't be rendered but you don't actually need to render here anyway)